packages/core/database/lib/schema/builder.js

```diff
@@ -295,7 +295,11 @@ const createHelpers = (db) => {
 
         const { object } = updatedColumn;
 
-        createColumn(tableBuilder, object).alter();
+        if (object.type === 'increments') {
+          createColumn(tableBuilder, { ...object, type: 'integer' }).alter();
```

Comment:
Question, when you want to update an `increments` column you are transforming it to `integer`?
Doesn't this drop the primary key from the column?

Reply:
An increment column is an integer column so it's not going to change the primary key (which is a constraint)